### PR TITLE
fix(filters): uuid fields are being parsed to an int

### PR DESCRIPTION
**Pull request by christiaanwesterbeek**

### CODE REVIEW

File: src/buildQuery.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ let typeMap: any
 let queryMap: QueryMap
 
 const mapType = (idType: any, value: string | number) =>
-  idType.name === 'String' ? value : parseInt(value as string, 10)
+  idType.name === 'String' ? value : idType.name === 'UUID' ? value : parseInt(value as string, 10)
```
Comment (benjie) [inline src/buildQuery.ts]:
```suggestion
  ['UUID', 'Uuid', 'String'].includes(idType.name) ? value : parseInt(value as string, 10)
```

:sparkles: legacy

(We used to use the lowercase form `Uuid`, and users can still opt into this, so just support both. Also simplified with this in mind.)

Comment (BowlingX) [reply src/buildQuery.ts]: Thank you! Makes sense :) We could also directly check the lowercased version's maybe instead:

`['uuid', 'string'].includes(idType.name.toLowerCase()) `

We check the type here: https://github.com/BowlingX/ra-postgraphile/blob/master/src/filters.ts#L18 again. So maybe it makes sense to introduce a normalize function for the names.

**Comment by BowlingX** [comment]
FYI. I adjusted circle-ci's config, I hope if you do another commit it will run the tests.

**Comment by BowlingX** [comment]
Please run `yarn prettify` once. There is a precommit hook that should prettify all staged files automatically.

**Comment by christiaanwesterbeek** [comment]
I'm not sure what to do. Add a test for the mapType method maybe so to increase coverage?

**Comment by BowlingX** [comment]
I will merge it for now :). We can add more tests on the go :).

**Pull request merged**